plugins/org.jkiss.dbeaver.model.sql/src/org/jkiss/dbeaver/model/sql/completion/SQLCompletionAnalyzer.java

```diff
@@ -1288,11 +1289,19 @@ private SQLCompletionProposalBase makeProposalsFromObject(
                             Statement sqlStatement = ((SQLQuery) request.getActiveQuery()).getStatement();
                             if (sqlStatement != null) {
                                 TablesNamesFinder namesFinder = new TablesNamesFinder() {
-                                    public void visit(Table table) {
+                                    public void visit(@Nullable Table table) {
                                         if (table != null && table.getAlias() != null && table.getAlias().getName() != null) {
                                             aliases.add(table.getAlias().getName().toLowerCase(Locale.ENGLISH));
                                         }
                                     }
+                                    
+                                    public void visit(@Nullable CreateView createView) {
+                                        if (createView != null && createView.getView().getAlias() != null 
+                                            && createView.getView().getName() != null
```

Comment:
You can invert this statement and perform an early exit to avoid having this cumbersome condition spilled over several lines.

Reply:
Not sure that it's better